src/components/Apps/AccountListTable/AccountList.vue

```diff
@@ -229,12 +240,20 @@ export default {
             }
           },
           asset: {
+            minWidth: '100px',
             formatter: function(row) {
               return row.asset.name
             }
           },
           username: {
-            width: '120px'
+            minWidth: '60px',
+            formatter: function(row) {
+              if (row.ds && row.ds['domain_name']) {
+                return `${row.username}@${row.ds['domain_name']}`
+              } else {
+                return row.username
+              }
+            }
           },
           secret_type: {
             formatter: function(row) {
```

Comment:
I do not have access to the code you shared since it is being deleted every few days. Please share the code with me so I could analyze its discrepancies between different versions and suggest optimizations accordingly.